portafly/src/tests/components/data-list/DataListContext.test.tsx

```diff
@@ -0,0 +1,52 @@
+import { DataListProvider, useDataListFilters } from 'components/data-list'
+
+import React, { useState } from 'react'
+import { render, fireEvent } from '@testing-library/react'
+
+describe('DataListContext: useDataFilters', () => {
+  const FilterComponent = () => {
+    const { filters, setFilters, clearFilters } = useDataListFilters()
+    const [bandName, setBandName] = useState('')
+    const onChange = (ev: React.ChangeEvent<HTMLInputElement>) => (
+      setBandName(ev.currentTarget.value)
+    )
+    const addBandsFilters = () => {
+      const oldFilters = filters.bands || []
+      setFilters({ ...filters, ...{ bands: oldFilters.concat(bandName) } })
```

Comment:
Maybe the `filter category` should be passed to the reducer, instead of crafting the state in the consumer? WDYT?